### DISCUSSION TITLE
HintAssist floating hint position

### DISCUSSION THREAD

**Discussion by jbooth88** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#3097] [Q&A] [answered]
Is it possible to enforce the MaterialDesignFloatingHintTextBox template SmartHint object to IsHintInFloatingPosition=true regardless of content or keyboard focus?

**Comment by nicolaihenriksen** [comment, marked as answer]
I don't think you can do that at the moment. Looking at [this part](https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/blob/ef7214b6fdecde27dd7c04192a998483c2391082/MaterialDesignThemes.Wpf/SmartHint.cs#L210-L213) of the `SmartHint`, it seems that it very much depends on focus and non-emptyness. This code sets the visual state which triggers the transition (i.e. animation) that levitates/floats the hint.
